webpack.dev.js

```diff
@@ -0,0 +1,57 @@
+const merge = require('webpack-merge');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+const baseConfig = require('./webpack.base.js');
+
+module.exports = merge(baseConfig, {
+  devtool: 'inline-source-map',
+
+  devServer: {
+    historyApiFallback: true,
+    hot: true,
+    inline: true,
+    host: 'localhost',
+    port: 8000,
+    proxy: {
+      '/api/*': {
+        target: 'http://localhost:3000/',
+        secure: false,
+        changeOrigin: true,
+      },
+    },
+  },
+
+  module: {
+    rules: [
+      {
+        test: /\.scss$/,
+        exclude: /node_modules/,
+        use: [
+          {
+            loader: 'style-loader',
+          },
+          {
+            loader: 'css-loader',
+            options: {
+              sourceMap: true,
+            },
+          },
+          {
+            loader: 'sass-loader',
+            options: {
+              sourceMap: true,
+            },
+          }
+        ],
+      },
+    ],
+  },
+
+  plugins: [
+    new HtmlWebpackPlugin({
+      title: 'Authors Haven',
+      template: './client/index.html'
+    }),
+  ],
```

Comment:
include the `HotModuleReplacementPlugin`

Reply:
Feedback has been implemented